### PR TITLE
Gaia guardian tweaks

### PR DESCRIPTION
**Pull request by Hubry**
Submitting for feedback, mostly.
cc @williewillus @Alwinfy

* a bit of anticheese
* more loot: dice of fate give spirits and some more random stuff once out of relics, because people dislike having to grind the guardian (not our fault, really, but modpack authors sometimes don't playtest their own grind)
* a bit of niceties: nerfing health in multiplayer by a bunch, killing pixies and clearing wither on death of the guardian, bumping the damage cap, and triggering win advancements for everyone involved, not just the true killer

Any of this anticheese is overkill? Anything to tweak?

### CODE REVIEW

File: src/main/java/vazkii/botania/common/item/relic/ItemDice.java [hunk 1]
```diff
@@ -69,14 +75,31 @@ public ItemDice(Properties props) {
...
-				player.sendMessage(new TranslationTextComponent("botaniamisc.dudDiceRoll", world.rand.nextInt(6) + 1).mergeStyle(TextFormatting.DARK_GREEN), Util.DUMMY_UUID);
-				stack.shrink(1);
-				return ActionResult.resultSuccess(stack);
-			} else {
+			if (!possible.isEmpty()) {
 				int relic = possible.get(world.rand.nextInt(possible.size()));
 				player.sendMessage(new TranslationTextComponent("botaniamisc.diceRoll", relic + 1).mergeStyle(TextFormatting.DARK_GREEN), Util.DUMMY_UUID);
 				return ActionResult.resultSuccess(new ItemStack(getRelics()[relic]));
+			} else {
+				int roll = world.rand.nextInt(6) + 1;
```
Comment (williewillus) [inline src/main/java/vazkii/botania/common/item/relic/ItemDice.java:83]: hardcoding the number of tables here is meh. Can we just have a dice/bonus_roll table that dispatches to the 6 subtables using weighted entries?

**Review by williewillus — COMMENTED** [review]
looks good besides the comment

File: src/main/java/vazkii/botania/common/item/relic/ItemDice.java (hunk 1 above)
Comment (Hubry) [reply src/main/java/vazkii/botania/common/item/relic/ItemDice.java:83]: I wanted to actually tie the roll to the table used, a minor and inconsequential detail, I guess I'll cut that though.

Comment (williewillus) [reply src/main/java/vazkii/botania/common/item/relic/ItemDice.java:83]: o lmao

**Pull request merged**